src/styles/index.scss

```diff
@@ -1,3 +1,3 @@
-body {
-  margin: 0;
-}
+@import '../utils/veriables';
```

Comment:
There is a typo in the import path. It should be '../utils/variables' instead of '../utils/veriables'.